lib/kitchen/directions/bake_chapter_references/v1.rb

```diff
@@ -2,11 +2,11 @@
 
 module Kitchen::Directions::BakeChapterReferences
   class V1
-    def bake(chapter:, metadata_source:, uuid_prefix: '.', klass: 'references')
-      bake_page_references(page: chapter.introduction_page)
+    def bake(chapter:, metadata_source:, uuid_prefix: '.', klass: 'references', module_title: true)
+      bake_page_references(page: chapter.introduction_page, module_title: module_title)
```

Comment:
see other comment, but could `MoveCustomSectionToEocContainer` also be used here instead of `BakeChapterReferences`? it seems like with `module_title=false`, all this does to the content of the section is just removing the redundant title. not sure I'm reading this right